src/index.html

```diff
@@ -69,6 +74,9 @@ <h5 class="modal-title" id="exportModalLabel">Export</h5>
     <div class="row" id="edit-area">
         <div class="col-12" id="editor"></div>
     </div>
+    <div class="row mt-3" id="eval-result-div">
+        <pre id="eval-result" class="form-control"></pre>
+    </div>
     <div class="row mt-3" id="env-div">
         <textarea id="env-data" class="form-control" placeholder="Environment data"></textarea>
```

Comment:
Maybe as a future TODO/GH issue, could include an example environment to show the structure of the environment data

Reply:
@alancai98 you mean as part of the environment text area pre-filled or somewhere else?

Reply:
https://github.com/partiql/partiql-rust-playground/issues/7

Reply:
Could be pre-filled or if there were more text instructions on the playground page, we could include the sample environment there.

Something similar to what's done for:
- Amazon Ion: https://amazon-ion.github.io/ion-docs/index.html
- Amazon Ion Schema: https://amazon-ion.github.io/ion-schema/sandbox